GiniVision/Classes/Core/GiniQRCodeDocument.swift

```diff
@@ -17,6 +17,7 @@ import Foundation
     }()
     public var isReviewable: Bool = false
     public var isImported: Bool = false
+    public var error: Error?
```

Comment:
Again, this error... It's also duplicated now quite a few times. I know the error is just a protocol and this declaration has to be repeated here, but... it's by not 5 times.

Reply:
But I don't get this, it is part of a protocol as it is `type` or `data`.

Reply:
It's not a big deal. I just don't like properties in a protocol because you just have to repeat the same thing in all classes implementing it.

Reply:
I don't see it as an issue, since you conform to a common interface/protocol. The idea would be to add it only in the `GiniVisionDocument`, but it doesn't allow stored properties (there is a workaround to add stored properties to a protocol, but I would rather not to use it).